cl/corpus_importer/tests.py

```diff
@@ -285,6 +285,33 @@ def test_get_fed_court_object_from_string(self):
                 got,
             )
 
+    def test_get_appellate_court_object_from_string(self):
+        """Can we get the correct federal appellate courts?"""
+
+        pairs = (
+            {
+                # FJC data appears to have a space between U and S.
```

Comment:
This comment is wrong, isn't it? It's the opposite and I had it wrong when I sketched out this code?

Reply:
I just reworded this.